backend/src/main/java/gov/cdc/usds/simplereport/db/model/DeviceType.java

```diff
@@ -50,11 +52,15 @@ public class DeviceType extends EternalAuditedEntity {
   @Column(nullable = false)
   private int testLength;
 
-  @JsonIgnore
   /** This relationship is necessary for DeviceTypeRepository.findAllByTestOrdersInternalIdIn */
+  @JsonIgnore
   @OneToMany(mappedBy = "deviceType", fetch = FetchType.LAZY)
   List<TestOrder> testOrders;
 
+  @JsonIgnore
+  @OneToMany(mappedBy = "deviceType", cascade = CascadeType.ALL, orphanRemoval = true)
+  List<DeviceTestPerformedLoincCode> deviceTestPerformedLoincCodeList = new ArrayList<>();
```

Comment:
This needs to have FetchType.LAZY

Reply:
One to many default is lazy
<img width="689" alt="image" src="https://user-images.githubusercontent.com/10108172/214953979-a0541415-0201-4983-8993-4de1ee9419f0.png">